### DISCUSSION TITLE
Server not in list

### DISCUSSION THREAD

**Discussion by terrachild** [etlegacy/etlegacy#2053] [Q&A] [answered]
I start up ET Legacy as a dedicated internet server, but when people look for my server in game on the list it doesn't show up, and also if I give them the listed ip in the console for my server they see: 'connecting' with a timer, but they never get in.  Am I doing something wrong?

**Comment by rafal1137** [comment, marked as answer]
In order to list your server on the browser make sure you have done PortForwarding on your router.
I suggest you to open a range of ports from `27950-27969` and those should be UDP ones.
